tekton/tasks/create-cluster.yaml

```
@@ -41,3 +48,59 @@ spec:
       --output $(workspaces.cluster.path)
 
       cat $(workspaces.cluster.path)/cluster-id > $(results.cluster-id.path)
+
+  - name: create-cluster-kubectl-gs
+    image: quay.io/giantswarm/kubectl-gs:1.39.1
+    volumeMounts:
+      - name: endpoints-config
+        mountPath: /etc/endpoints-config
+      - name: kubeconfig
+        mountPath: /etc/kubeconfig
+    script: |
+      #! /bin/sh
+
+      if [ "$(params.release-id)" != "20.0.0" ]
+      then
+        echo "Release $(params.release-id) is not a CAPI release"
+        exit 0
+      fi
+
+      # Generate template for control plane.
+      kubectl-gs template cluster \
+        --provider azure \
+        --release 20.0.0 \
+        --owner giantswarm \
+        --output $(workspaces.cluster.path)/cluster.yaml
+
+      # Extract cluster name from template.
+      cluster_id="$(cat $(workspaces.cluster.path)/cluster.yaml | grep cluster.x-k8s.io/cluster-name | head -n1 | cut -d'"' -f4)"
+
+      echo -n "$(cluster_id)" > $(results.cluster-id.path)
+
+      kubectl-gs template nodepool \
+        --provider azure \
+        --release 20.0.0 \
+        --owner giantswarm \
+        --cluster-name "$(cluster_id)" \
+        --output /tmp/nodepool.yaml
+
+      cat /tmp/nodepool.yaml >>$(workspaces.cluster.path)/cluster.yaml
+
+  - name: apply-capi-manifest
+    image: bitnami/kubectl:1.18
```

```suggestion
    image: quay.io/giantswarm/kubectl:1.18.19
```
(docker hub would be better yes but I checked and that image isn't synced there currently)